crowdsourcing/serializers/user.py

```diff
@@ -135,15 +137,18 @@ def create(self, **kwargs):
         user_profile = models.UserProfile()
         user_profile.user = user
         user_profile.save()
-        worker = models.Worker()
-        worker.profile = user_profile
-        worker.alias = username
-        worker.save()
-
-        requester = models.Requester()
-        requester.profile = user_profile
-        requester.alias = username
-        requester.save()
+
+        if 'is_worker' not in self.validated_data or self.validated_data['is_worker']:
```

Comment:
Yep, creating profiles was not really supposed to be here, by default a user is neither I know now we are making him both ;)